android/app/src/androidTest/java/org/fossasia/openevent/DateTest.java

```diff
@@ -0,0 +1,362 @@
+package org.fossasia.openevent;
+
+import org.fossasia.openevent.utils.ISO8601Date;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Created by arpitdec5 on 02-06-2017.
+ */
+
+public class DateTest {
+
+    @Test
+    public void shouldConvertSingaporeDateStringToUTCDateString() {
```

Comment:
basically you are trying to say keep all the stuff in one method right?
one testing all 4 strings with specified timezone and eventTimeZone and the the single dateString testing with different timeZones?